geo-location.html

```diff
@@ -43,141 +40,149 @@
       }
     </style>
   </template>
-  <script>
-    (function() {
-      var id_ = null;
+</dom-module>
+<script>
+  (function() {
+    var id_ = null;
 
-      /**
-       * Fired when the Geolocation API returns an error.
-       *
-       * @event geo-error
-       * @param {Object} detail
-       * @param {boolean} detail.error The error message.
-       */
+    Polymer({
 
-      /**
-       * Fired when the Geolocation API returns a position result.
-       *
-       * @event geo-response
-       * @param {Object} detail
-       *   @param {Position} position The raw position object returned by the Geolocation API.
-       *   @param {Number} detail.latitude Latitude of the current position.
-       *   @param {Number} detail.longitude Longitude of the current position.
-       */
+      is: 'geo-location',
 
-      Polymer({
-
-        publish: {
-          /**
-           * The latitude of the current position.
-           *
-           * @attribute latitude
-           * @type Number
-           * @default null
-           */
-          latitude: {value: null, reflect: true},
-
-          /**
-           * The longitude of the current position.
-           *
-           * @attribute longitude
-           * @type Number
-           * @default null
-           */
-          longitude: {value: null, reflect: true}
+      properties: {
+        /**
+         * The latitude of the current position.
+         */
+        latitude: {
```

Comment:
Should I kill reflectToAttribute for these? I used it for debugging, it is not that useful